src/vivarium_nih_us_cvd/components/effects.py

```diff
@@ -107,25 +98,13 @@ def _polypill_sbp_adherence_modifier(
 class PAFCalculationRiskEffect(RiskEffect):
     """Risk effect component for calculating PAFs"""
 
-    def __repr__(self):
-        return f"PAFCalculationRiskEffect(risk={self.risk}, target={self.target})"
-
-    ##############
-    # Properties #
-    ##############
-
-    @property
-    def name(self) -> str:
-        return f"paf_calculation_risk_effect.{self.risk}.{self.target}"
+    def get_population_attributable_fraction_source(
```

Comment:
So the purpose of this component is to behave like a standard risk effect accept it skips the PAF stuff?

Reply:
It needs to calculate risk effects, which are then used by the PAF calculation observer.